shell/common/extensions/api/tabs.json

```diff
@@ -3,7 +3,14 @@
     "namespace": "tabs",
     "description": "Use the <code>chrome.tabs</code> API to interact with the browser's tab system. You can use this API to create, modify, and rearrange tabs in the browser.",
     "types": [
-      { "id": "MutedInfoReason",
+      {
+        "id": "TabStatus",
+        "type": "string",
+        "enum": ["unloaded", "loading", "complete"],
+        "description": "The tab's loading status."
+      },
+      { 
+        "id": "MutedInfoReason",
```

Comment:
Before we merge this, I'd like to sync parts of this file from upstream to ensure the schema is up to date.